.gitignore

```diff
@@ -11,3 +11,4 @@
 CMakeLists.txt
 .idea/
 cmake-build-debug/
+.vscode/*
```

Comment:
This change is ok, but should be in a separate commit

Reply:
It is

Reply:
Oops, sorry!